Tests/TestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace JakubZapletal\Component\BankStatement\Tests;
+
+
+class TestCase extends \PHPUnit_Framework_TestCase
+{
+}
+
+/*
+ * Convers E_RECOVERABLE_ERROR to exception.
+ * - used for tests relying on type hinting exceptions
+ */
+set_error_handler(function($errno, $errstr, $errfile, $errline) {
```

Comment:
Current tests doesn't work for me (at least those named `test...Exception`) without this workaround. They rely on `RuntimeException` being thrown by type hinted methods, but I got errors instead of them.

For example:

    1) JakubZapletal\Component\BankStatement\Tests\Parser\ABOParserTest::testParseFileObjectException
    Argument 1 passed to JakubZapletal\Component\BankStatement\Parser\ABOParser::parseFileObject() must be an instance of SplFileObject, array given

    /Parser/ABOParser.php:62
    /Tests/Parser/ABOParserTest.php:168

I think it is caused by different PHP versions. Which version do you use? I use 5.6.6.